packages/teams-js/src/public/runtime.ts

```diff
@@ -611,6 +616,8 @@ export const mapTeamsVersionToSupportedCapabilities: Record<string, Array<ICapab
       hostClientTypes: [HostClientType.android, HostClientType.ios, HostClientType.ipados, HostClientType.visionOS],
     },
   ],
+  //teamsMobileVersionLegacyForDeeplyNestedAuth 2.1.2 is for deeplyNestedAppAuth but
+  //doesn't have any new capability added into it.
 };
```

Comment:
We should use the exact object name as defined in runtime object to avoid ambiguity.  
```
'2.1.1': [
     // isDeeplyNestedAuthSupported: true (based on const legacyTeamsMobileVersionForDeeplyNestedAuth)
    {
      capability: { nestedAppAuth: {} },
      hostClientTypes: [HostClientType.android, HostClientType.ios, HostClientType.ipados, HostClientType.visionOS],
    },
  ],
```

Reply:
Sorry, the version number is incorrect. I meant to say, can we add a placeholder here and then add a comment that isDeeplyNestedAuthSupported is true? 

For example:
```
  '2.1.2': [
    // isDeeplyNestedAuthSupported: true (based on const legacyTeamsMobileVersionForDeeplyNestedAuth)
  ],

```